서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,7 @@
+function solution(text1) {
```

Comment:
문제에서 제시한 seoul이라는 이름이 text1보다 좋은거 같아요
```suggestion
function solution(seoul) {
```